ui/components/ui/typography/typography.js

```diff
@@ -13,24 +13,92 @@ import Box, { MultipleSizes } from '../box';
 
 const { H6, H7, H8, H9 } = TYPOGRAPHY;
 
+export const ValidColors = [
```

Comment:
Not sure if I should export this if the only other place it is being used is in `typography.stories.js` to help with docs? It will safe repeating the same array and should help keep one source of truth